pyiron/table/datamining.py

```diff
@@ -802,6 +802,7 @@ def always_true_pandas(job_table):
         pandas.Series: A series of True values
 
     """
+    import pandas
```

Comment:
Why are you reimporting pandas?

Reply:
Exactly - when reloading the function from the HDF5 file all import statements have to be included in the same function. This broke my calculations.

Reply:
In that case, it's better to import `pandas.Series`  as `Series` in the function. This avoids the unpythonic reimporting of the `pandas` module.